client/my-sites/pages/main.jsx

```diff
@@ -52,8 +56,26 @@ class PagesMain extends React.Component {
 	}
 
 	render() {
-		const { doSearch, search, translate } = this.props;
-		const status = this.props.status || 'published';
+		const { doSearch, search, translate, siteId } = this.props;
+		let status = this.props.status || '';
+		let analyticsPageTitle = 'Pages';
+		let baseAnalyticsPath = '/pages';
+
+		if ( status ) {
+			baseAnalyticsPath += '/:status';
+		}
+
+		if ( siteId ) {
+			baseAnalyticsPath += '/:site';
+		}
+
+		if ( status.length ) {
+			analyticsPageTitle += ' > ' + titlecase( status );
+		} else {
+			analyticsPageTitle += ' > Published';
+			status = 'published';
+		}
```

Comment:
I think we could shorten this a bit:

```es6
const analyticsPageTitle = `Pages > ${ status.length ? titlecase( status ) : 'Published' }`;
const baseAnalyticsPath = `/pages/${ status.length ? ':status' : ':site' }`;
```

Reply:
I find the current version of `baseAnalyticsPath` more readable. `analyticsPageTitle` is a better candidate, but I'm a bit concerned with juggling `status` and `this.props.status` which have slightly different meanings.

Reply:
Either way is fine. I just get triggered by `let`. 😄

Reply:
If we want to be tidy it further, another option is also creating a function for just this and early returning eg:
```
getAnalyticsPath = function() {
    const { status, siteId } = this.props;
    if ( status && siteId ) {
        return `/pages/${ status }/:site`;
	}
    if ( siteId ) {
		return `/pages/:site`;
	}
    return '/pages';
}
getAnalyticsPageTitle = function() {
//...
}
```

Reply:
Sounds good. Refactored in 61c027f45da27d3a611472711652da613a4dc8be.